### PR TITLE
Implemented gethostname() with syscall number: 125

### PR DESCRIPTION
**Pull request by youngfalcon**

### CODE REVIEW

File: seattlelib/lind_net_calls.py [hunk 1]
```diff
@@ -1714,7 +1714,34 @@ def epoll_wait_syscall(epfd, maxevents, timeout):
     return nepoll_return, epoll_return
 
   NET_CALL_DICTIONARY["epoll_wait_syscall"] = epoll_wait_syscall
+  
+  # int gethostname(char *name, size_t len);
```
Comment (rennergade) [inline seattlelib/lind_net_calls.py:1718]: I think you accidentally left this in here as a reference when you were coding.

**Review by rennergade — COMMENTED** [review]
Other than the extraneous comment, looks good!

**Comment by youngfalcon** [comment]
It was not an accident. Some syscalls in the file are accompanied by such comments depicting the system calls they are implementing. (See socket_syscall for example)

**Comment by rennergade** [comment]
Ah, okay. Seems fine then!

**Pull request merged**